ir/shared/src/main/scala/org/scalajs/ir/Trees.scala

```diff
@@ -897,6 +897,15 @@ object Trees {
      */
     def isValidJSGlobalRefName(name: String): Boolean =
       isJSIdentifierName(name) && !ReservedJSIdentifierNames.contains(name)
+
+    /** The value of the JavaScript that is an alias to `this`
+     *  at the top-level of the generated file.
+     *
+     *  `scala.scalajs.js.special.fileLevelThis` should be compiled to
+     *  `JSGlobalRef("$fileLevelThis")`, and the backends are expected to
+     *  generate `const $fileLevelThis = this` at the top-level of the file.
+     */
+    final val FileLevelThis = "$fileLevelThis"
```

Comment:
As mentioned before:
```suggestion
    final val FileLevelThis = "this"
```